ArcadiaParties-SPA/src/app/user/user-info/user-info.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="flex-container">
+  <mat-card>
+    <mat-card-title style="text-align:center">Your information</mat-card-title>
+    <mat-tab-group mat-stretch-tabs>
+      <mat-tab label="General">
+        <mat-card-content>Identity: {{ (user$ | async).identity }} </mat-card-content>
```

Comment:
There are two problems here:
1) With using of async pipe you added 6 subscribes in one page. It is not very much, but if possible such things should be avoided.
2) And more important is if loading of user takes some time , your page will throw an exception `Cannot read property 'firstName' of null`.

To deal with it you can add `ngIf` directive to `mat-tab-group` with an alternative template and storing of conditional result.

Reply:
NgIf added